roles/openshift_health_checker/openshift_checks/etcd_traffic.py

```diff
@@ -0,0 +1,59 @@
+"""Module that scans journalctl for messages caused as a symptom of increased etcd traffic"""
+
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class EtcdTraffic(OpenShiftCheck):
+    """Check if host is being affected by an increase in etcd traffic."""
+
+    name = "etcd_traffic"
+    tags = ["health", "etcd"]
+
+    @classmethod
+    def is_active(cls, task_vars):
+        """Skip hosts that do not have etcd or masters in their group names."""
+        master_count = get_var(task_vars, "openshift", "master", "master_count", default=1)
+        group_names = get_var(task_vars, "group_names", default=[])
+        active = "masters" in group_names
+        if int(master_count) > 1:
+            active = "etcd" in group_names
```

Comment:
Late to the party, but yes - I think rather than try to figure out exactly where etcd is running, which is looking more complicated all the time, just use the shotgun approach and look at both the master and etcd journals on any master or etcd host. It's only slightly inefficient and it's much simpler.